README.md

```diff
@@ -30,6 +30,10 @@ var App = SecretStack({ appKey: '1KHLiKZvAvjbY1ziZEHMXawbCEIM6qwjCDm3VYRan/s=' }
 
 var app = App(config)
 ```
+Added talk.js and listen.js that can run on 2 different machines.
+Usage: node listen.js
+Then copy the key it prints and the host IP on which you ran listen.js and put it in talk.js
+Usage: node talk.js
```

Comment:
the markdown here doesn't display well, e.g. https://github.com/nuttwork/secret-stack/blob/example/README.md

might be better written like:

```markdown
See also: [`talk.js`](./examples/talk.js) and [`listen.js`](./examples/listen.js)
```

Where the rest of the documentation could be in those files.